code/src/java/pcgen/cdom/content/LevelCommandFactory.java

```diff
@@ -148,11 +148,7 @@ public int compareTo(LevelCommandFactory other)
 			{
 				return 0;
 			}
-			if (levels == other.levels)
```

Comment:
If these are equal, compareTo will return 0 anyway (doesn't seem to imply there should be a special case for reference equality)